.github/workflows/build-and-test.yml

```diff
@@ -230,14 +229,18 @@ jobs:
           - 0  # use CTK wheels
         runner:
           - default
+        exclude:
+          # To test this combo would require nontrivial installation steps.
+          - cuda-version: "12.0.1"
+            local-ctk: 0
```

Comment:
For self-consistency, I'd either

* make this `local-ctk: "0"` (and `"1"` a few lines down), or
* or change the existing `matrix.local-ctk == '1'` to `matrix.local-ctk == 1` below.

Reply:
oh interesting... I dunno what I was thinking when I added `local-ctk`.

Reply:
ooohhhh @rwgk I recalled now. When `local-ctk` being passed to the test workflow, it's type-cast to string:
https://github.com/NVIDIA/cuda-python/blob/4aa28747745f29f7a6e86c391a1d31a2d357e739/.github/workflows/test-wheel.yml#L18-L20
and it's needed because while we have strings and numbers in YAML, we don't have that in shell scripts (at least how the type conversion works when accessing these YAML variables in shell via `${{ local-ctk }}` is unclear to me), so it was simpler for me to treat them as string everywhere.